jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh

```diff
@@ -20,7 +20,7 @@ set -e -x
 terra_jupyter_python="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:1.0.4"
 terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:2.0.5"
 terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:2.0.5"
-terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:1.0.8"
+terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:1.0.9"
 terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.0.8"
 terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.0.12"
```

Comment:
Any chance the AoU 2.0.13 image can hop on this train (Dataproc and GCE)? Not urgent, so if the Hail fix is time sensitive, please ignore.

https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-aou/CHANGELOG.md#2013---2022-03-18

Reply:
Yep, we can bump this as well. I'm noticing that we store an old image on the leo image as well, should we also bump this version up to 2.0.12?

Reply:
2.0.11 is in active use, so please keep that as the "old" version. 2.0.12 did not wind up being used by us, so no need to cache it. Thank you!